src/D3MMom.sol

```diff
@@ -16,16 +16,16 @@
 
 pragma solidity 0.6.12;
```

Comment:
Would recommend `pragma solidity >=0.6.12;` (or `>=0.8.13`) everywhere for parity with dss and other modules and to not lock future implementations and projects that depend on this to a specific solc version.

Reply:
Could also get away with `^0.6.12` but it's more restrictive than the other one.

Reply:
added here: c0fc144